website/docs/en/plugins/rspack/copy-rspack-plugin.mdx

```diff
@@ -0,0 +1,177 @@
+import { Table } from '@builtIns';
+import WebpackLicense from '../../../../components/webpack-license';
+import { ApiMeta } from '../../../../components/ApiMeta.tsx';
+
+<WebpackLicense from="https://webpack.js.org/plugins/copy-webpack-plugin/#root" />
+
+# CopyRspackPlugin
+
+<ApiMeta addedVersion={'0.3.3'} />
```

Comment:
Can we remove the `New` label of 0.3.x or 0.4.x versions, as they are not very "new"..

![image](https://github.com/web-infra-dev/rspack/assets/7237365/986228b5-790b-4f7f-8b85-c0934689c891)